frontend/lib/ui/InputGrid/InputGrid.module.css

```diff
@@ -46,7 +46,7 @@
         border-right: 1px solid var(--bg-gray-darkest);
         padding: 0;
         input {
-          font-family: var(--font-number);
+          font-family: var(--font-number) var(--font-text);
```

Comment:
@praseodym Wouldn't it be a better idea to set the fallback font in the `--font-number` variable? That way, we can define a fallback that is monospaced, just like the Geist Mono font, and retain as much of the original layout/design choices a possible. 

if we want to define a generic 'monospaced' fallback, we should do it like this : https://stackoverflow.com/questions/38781089/font-family-monospace-monospace

Reply:
Fixed in 4cd653fd1cf344c4f6d510df50fe6b21df634e74.